src/1_utils.js

```diff
@@ -38,7 +38,7 @@ utils.httpMethod = {
  * }} */
 utils.resource;
 
-/** @typedef {{listener: function(string):null, event: string}} */
+/** @typedef {{listener: function(string, Object):null, event: string}} */
```

Comment:
We could additionally describe the object as:

+/** @typedef {{listener: function(string, Object):null, event: string, data: object}} */